src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -72,6 +80,17 @@ export class BaseEditCourseForm extends React.Component {
     }
   }
 
+  resetOpenCourseRuns() {
```

Comment:
Nit: rename to `closeOpenCourseRuns`. Reset makes it sound like resetting data or something and this doesn't do that.